public/app/features/admin/AdminFeatureTogglesPage.tsx

```diff
@@ -0,0 +1,57 @@
+import { createApi, BaseQueryFn } from '@reduxjs/toolkit/query/react';
+import React from 'react';
+import { lastValueFrom } from 'rxjs';
+
+import { getBackendSrv } from '@grafana/runtime';
+import { Page } from 'app/core/components/Page/Page';
+
+import { AdminFeatureTogglesTable } from './AdminFeatureTogglesTable';
+
+type FeatureToggle = {
+  name: string;
+  enabled: boolean;
+  description: string;
+};
```

Comment:
You could move `FeatureToggle` to `adminFeatureTogglesAPI.ts` and export it,  so you can import and reuse it in this file and in the table file.